fjord/manage_utils.py

```diff
@@ -172,25 +173,54 @@ def monkeypatch():
     admin.site = AdminSitePlus()
     admin.site.login = login_required(anonymous_csrf(admin.site.login))
 
-    # Monkey-patch django forms to avoid having to use Jinja2's |safe
-    # everywhere.
-    import jingo.monkey
-    jingo.monkey.patch()
-
     # Monkey-patch Django's csrf_protect decorator to use
-    # session-based CSRF tokens.
+    # session-based CSRF tokens
     import session_csrf
     session_csrf.monkeypatch()
 
-    from jingo import load_helpers
-    load_helpers()
-
     logging.debug('Note: monkeypatches executed in %s' % __file__)
 
-    # Prevent it from being run again later.
+    # Prevent it from being run again later
     _has_patched = True
 
 
+def monkeypatch_render():
+    import django.shortcuts
+
+    def more_info(fun):
+        """Django's render shortcut, but captures information for testing
+
+        When using Django's render shortcut with Jinja2 templates, none of
+        the information is captured and thus you can't use it for testing.
+
+        This alleviates that somewhat by capturing some of the information
+        allowing you to test it.
+
+        Caveats:
+
+        * it does *not* capture all the Jinja2 templates used to render.
+        Only the topmost one requested by the render() function.
+
+        """
+        @wraps(fun)
+        def _more_info(request, template_name, *args, **kwargs):
+            resp = fun(request, template_name, *args, **kwargs)
+
+            # If we're in a test situation, then capture it all
```

Comment:
Umm... Yeah. So this code doesn't affect whether it kicks on or not, but rather the code here:

https://github.com/mozilla/fjord/pull/684/files#diff-77b722d1cd2658a4eda87c2b19b62ca3R27

I'll nix this comment since it seems weird here. I'll add a docstring to `monkeypatch_render` that says only to use it for test contexts.